pm/sendermonitor.go

```diff
@@ -25,6 +25,8 @@ var unixNow = func() int64 {
 	return time.Now().Unix()
 }
 
+type checkTxError error
```

Comment:
```suggestion
type errCheckTx error
```

To match the error naming convention used elsewhere in this package.